packages/@okta/vuepress-site/docs/reference/api/org/index.md

```diff
@@ -0,0 +1,857 @@
+
+---
+title: Org
+category: management
+---
+
+# Org API
+
+<ApiLifecycle access="ea" />
+
+The Okta Org API provides operations to manage your org account settings such as contact information, granting Okta Support access, and more.
+
+## Get Org Settings
+
+<ApiOperation method="get" url="/api/v1/org" />
+
+Gets your Organization's current settings.
+
+### Response parameters
```

Comment:
Please ensure that this PR has sections that conform to the template. Currently it does not. In many different ways.

Reply:
Done